src/common/src/lib.rs

```diff
@@ -59,3 +59,5 @@ pub mod test_prelude {
     pub use super::array::{DataChunkTestExt, StreamChunkTestExt};
     pub use super::catalog::test_utils::ColumnDescTestExt;
 }
+
+pub const RW_VERSION: &str = "v0.1.15";
```

Comment:
Better to be generated by cargo or whatever. BTW our version in main branch is actually `0.2.0-alpha` 😄

Reply:
Furthermore, it would be great to include the commit ID (the SHA256 hash) in the output as well.